### PR TITLE
fix(docs-infra): fix top menu item clickable area

### LINKED ISSUE #27618: angular.io top nav item not clickable everywhere
# 📚 Docs or angular.io bug report

### Description

The menu items of the top nav bar in angular.io have a large clickable area, but the click has no effect if clicking outside of the region corresponding to the focused region.

## 🔬 Minimal Reproduction
clicking "events", stays on "docs"
docs1
docs2
docs4

clicking events inside the area corresponding to focused size, changing to "events"
docs3

### What's the affected URL?**
https://angular.io/

### Browser info
Google Chrome 67

### PR DESCRIPTION
**Pull request by macjohnny**
the clickable region of the top menu item is expanded beyond the focused area, so the clickable area is spans the entire height of the navigation

fixes #27618 

follow-up of #27620

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Currently, only the area with visible hover state of a menu top item is clickable.

Issue Number: #27618

## What is the new behavior?
The clickable area of the menu top item fits the entire height of the navigation bar.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/src/styles/1-layouts/_top-menu.scss [hunk 1]
```diff
@@ -147,13 +147,17 @@ aio-top-menu {
...
   a.nav-link {
     margin: 0;
-    padding: 8px 16px;
+    padding: 24px 0px;
     cursor: pointer;
-    border-radius: 4px;
-      
+    .nav-link-inner {
+      padding: 8px 16px;
+    }
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_top-menu.scss]: Can you leave empty lines between styles and nested blockes?

**Review by gkalpak — APPROVED** [review]
One minor comment. LGTM otherwise 👍 Thx!

File: aio/src/styles/1-layouts/_top-menu.scss (hunk 1 above)
Comment (macjohnny) [reply aio/src/styles/1-layouts/_top-menu.scss]: @gkalpak done

**Comment by gkalpak** [comment]
Thx for quick turnaround, @macjohnny.

+1 for adding the requested changes in a new commit (which makes them much easier to review), but the commit message now breaks linting (because it doesn't adhere to our commit message guidelines).
If you want to add a separate commit (for reviewing purposes), but have it squashed into the original one while merging, you can add a [fixup commit][1].

[1]: https://git-scm.com/docs/git-commit#git-commit---fixupltcommitgt

**Comment by macjohnny** [comment]
@gparlakov I rebased using fixup, so I hope it is alright, now.

**Comment by macjohnny** [comment]
@gparlakov I think now I got it right, the inital commit remains, while the fixup commit was added.
Should be good now

**Review by gkalpak — APPROVED** [review]
Looks great now, thx :smiley:

**Pull request closed without merging**